convert.py

```diff
@@ -0,0 +1,108 @@
+    print "***ADDITION TO THE FACTORS LISTED ABOVE, THE NUMBER OF INQUIRIES ON THE"
+    print "***CONSUMER'S CREDIT FILE HAS ADVERSELY AFFECTED THE CREDIT SCORE."
+    print "--------------------------------------------------------------------------------------"
+    print "T R A D E S"
+    print "SUBNAME       SUBCODE    OPENED      HIGHCRED   TERMS         MAXDELQ PAYPAT  1-12 MOP"
+    print "ACCOUNT#                 VERFIED     CREDLIM    PASTDUE       AMT-MOP PAYPAT 13-24"
+    print "ECOA COLLATRL/LOANTYPE   CLSD/PD     BALANCE    REMARKS                   MO 30/60/90"
+    tradeAccounts = root.find('.//ns:tradeAccounts', ns)
+    for account in tradeAccounts:
+        subname = str(account[2][2][1].text)
+        subcode = str(account[2][0].text) + " " + str(account[2][1].text)
+        opened = str(account[6].text)
+        highcred = str(account[12].text)
+        if highcred == 'None':
+            highcred = "???"
+        terms = "???"
+        if len(account[15]):
+            terms = str(account[15][0].text) + str(account[15][1].text)
+        accountnumber = str(account[4].text)
+        verified = str(account[7].text)
+        credlim = str(account[13].text)
+        if credlim == 'None':
+            credlim = "???"
+        pastdue = str(account[17].text)
+        if pastdue == 'None':
+            pastdue = "???"
+        ecoa = str(account[5].text)
+        loantype = str(account[3].text)
+        closed = str(account[8].text)
+        if closed == 'None':
+            closed = "???"
+        balance = str(account[11].text)
+        if balance == 'None':
+            balance = "???"
+        remarks = str(account[1].text)
+        if remarks == 'None':
+            remarks = "???"
+        startdate = "     ???"
+        paypattext = "     ???"
+        paypathistory = "  ???"
+        if len(account[18].find('ns:paymentPattern', ns)):
+            paypat = account[18].find('ns:paymentPattern', ns)
+            startdate = str(paypat[0].text)
+            paypattext = str(paypat[1].text)
+        if len(account[18].find('ns:historicalCounters', ns)):
+            history = account[18].find('ns:historicalCounters', ns)
+            paypathistory = str(history[0].text) + " " + str(history[1].text) + "/" + str(history[2].text) + "/" + str(history[3].text)
+        print subname.ljust(14) + subcode.ljust(11) + opened.ljust(12) + highcred.ljust(11) + terms.ljust(14) + " ".ljust(8) + startdate
+        print accountnumber.ljust(25) + verified.ljust(12) + credlim.ljust(11) + pastdue.ljust(14) + " ".ljust(8) + paypattext
+        print ecoa[0].ljust(5) + loantype.ljust(20) + closed.ljust(12) + balance.ljust(11) + remarks.ljust(25) + paypathistory
+        print ""
+    print "--------------------------------------------------------------------------------------"
+    print "I N Q U I R I E S"
+    print "DATE    SUBCODE     SUBNAME     TYPE     AMOUNT"
+    print ""
```

Comment:
add `( NO DATA )` to this section so that a reader is aware that this section is blank because there is no data in the xml.